mllib/src/main/scala/org/apache/spark/ml/feature/CountVectorizer.scala

```diff
@@ -241,7 +241,6 @@ class CountVectorizer @Since("1.5.0") (@Since("1.5.0") override val uid: String)
     }
     wordCounts.unpersist()
 
-    require(vocab.length > 0, "The vocabulary size should be > 0. Lower minDF as necessary.")
```

Comment:
Maybe it is also good to log something about the generated `vocab` is empty? This `require` isn't only a prohibition but also warns that `minDF` is set too high.

Reply:
Thanks for the comment. What is the appropriate log level for this? Warn?

Reply:
Added the warning